.github/actions/get-changed-gha-items/Get-GhaItemsFromFileList.ps1

```diff
@@ -4,15 +4,14 @@ param(
 
 # Filter actions based on files in action directory.
 [array]$actionFiles = $FileIncludeList | Where-Object -FilterScript {
-    try
-    {
-        (Get-Item $PSitem).Directory.GetFiles('action.yml').Count -gt 0 -or
-        (Get-Item $PSitem).Directory.GetFiles('action.yaml').Count -gt 0
-    }
-    catch
-    {
+    $itemDirectory = (Get-Item $PSItem).Directory.FullName
+    $isInGitHubDir = $itemDirectory -like '*/.github/*' -or $itemDirectory -eq '*/.github'
+    if (-not $isInGitHubDir) {
         return $false
     }
+
+    (Get-Item $PSitem).Directory.GetFiles('action.yml').Count -gt 0 -or
+    (Get-Item $PSitem).Directory.GetFiles('action.yaml').Count -gt 0
```

Comment:
This expression now is not used.

Reply:
I think I don't understand this comment.

That expression should be executed only for those files that are contained in the `.github` folder. Is it not working as intended?